addon/list-item-view-mixin.js

```diff
@@ -29,7 +29,7 @@ function positionElement() {
 
 export default Ember.Mixin.create({
   classNames: ['ember-list-item-view'],
-  style: '',
+  style: ('').htmlSafe(),
```

Comment:
this is a prototype extension, which means it wont work for users with those disabled, please use the long-form version